modules/kmods-installer/README.md

```diff
@@ -0,0 +1,31 @@
+# `kmods-installer` Module for Startingpoint
+
+The `kmods-installer` is a tool used for managing and installing kernel modules. It simplifies the installation of kernel modules, improving the capabilities of your system.
+
+List of all available kernel modules & versions/tags are here: https://github.com/ublue-os/akmods
+
+To use `kmods-installer` module, specify the akmods tag in Containerfile
+(line already exists at the 2nd last paragraph, you just need to adjust it):
+
+`# Starting with Fedora 39, the main image does not contain kmods
+COPY --from=ghcr.io/ublue-os/akmods:SPECIFY-YOUR-TAG-HERE /rpms /tmp/rpms`
+
+After that, specify kernel modules you wish to install in the `install:` section of your recipe/configuration file.
+
+## Example configuration
+```yaml
+type: kmods-installer
+
+install:
+    - openrazer
+    - openrgb
+    - v4l2loopback
+    - winesync
+```
+
+## Notes
+
+It should be used for Fedora 39 & above only.
+For Nvidia, use Universal Blue base Nvidia images + akmod builds with regular tag.
```

Comment:
> For Nvidia, use Universal Blue base Nvidia images + akmod builds with regular tag.

This line is a bit confusing, does this mean that Nvidia builds shouldn't use the module at all?  
This section could also be made to stand out more, maybe just by adding it as a warning right before the example config, reading something like:  
_Warning: only use this module with non-Nvidia builds based on Fedora 39 or older. <Explanation for why?> Only Universal Blue images are officially supported. Custom kernels are not supported, because..._

Reply:
This means that Nvidia users should use -nvidia builds as a base because it already contains needed nvidia kmod. 
So they can add the modules they want normally like the users who base from -main images.

I will try to rename this to be clearer or maybe even remove it completely as it can be seen as confusing.

Reply:
Addressed